List/src/main/java/com/list/ListManipulator.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ */
+
+package com.list;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ *
+ * @author vitor
+ */
+public class ListManipulator {
```

Comment:
`ListManipulator` doesn't seem very telling here, it just says that it is a class that manipulates lists. Perhaps we can call this `Grammar`. What do you think?